Undo chapter 2 notes duplication

lucio-cornejo opened the pull request:
Starting from [this](https://github.com/r4ds/bookclub-js4ds/blob/main/02_basic-features.Rmd#L437) line, the notes for chapter 2 have been duplicated, most likely as a result of me not merging before creating a pull request (won't happen again).

Commit:
create folder for js examples from the book

Commit:
add folder for images

Commit:
add notes for chapter 2

Commit:
add module demo for chapter 2

Commit:
include exercises in the main book

Commit:
add some exercises

Commit:
add all exercises for chapter 02

Commit:
fix typo

jonthegeek commented:
This has conflicts with the main branch. They resulted from creating a new PR for a branch that had already been merged. Once I merge your pull request, run `usethis::pr_finish()` to close out that branch and start anew.

I'll sort this out, and hopefully it will make sense eventually!

jonthegeek committed:
Fix merge conflict.

Merge commit 'b95aec0ff95e1373db75511c4e0d3a36fef04f70'

#Conflicts:
#	02_basic-features.Rmd

jonthegeek commented:
In general, I highly recommend that you simplify what you include in these. The book is the book. You're just making slides. There shouldn't be more than about 30 words on each slide (and usually far less).
